.github/workflows/container-reusable.yaml

```
@@ -70,13 +73,8 @@ jobs:
           registry: ${{ inputs.REGISTRY }}
           username: ${{ secrets.REGISTRY_LOGIN }}
           password: ${{ secrets.REGISTRY_PASSWORD }}
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v2
-
-      - name: install buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v2
+          erc: false
```

Spelling error
```suggestion
          ecr: false
```
https://github.com/docker/login-action#customizing

Also it's not used in second file, what is it's purpose?

---

It specifies whether the given registry is ECR ([Amazon ERC](https://aws.amazon.com/ecr/)) By default it's set to auto (in our case it's false). Looking at this again, I think this line can be deleted .